app/actions/local/channel.ts

```diff
@@ -303,21 +303,22 @@ export async function updateChannelInfoFromChannel(serverUrl: string, channel: C
 export async function updateLastPostAt(serverUrl: string, channelId: string, lastPostAt: number, prepareRecordsOnly = false) {
     try {
         const {database, operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
-        const member = await getMyChannel(database, channelId);
-        if (!member) {
+        const myChannel = await getMyChannel(database, channelId);
+        if (!myChannel) {
             return {error: 'not a member'};
         }
 
-        if (lastPostAt > member.lastPostAt) {
-            member.prepareUpdate((m) => {
+        if (lastPostAt > myChannel.lastPostAt) {
+            myChannel.resetPreparedState();
```

Comment:
Why is this hack needed?

Reply:
Becsuse qe updste lastPotAt in one prepare statement snd then lastFetchedAt in another prepare statement